benchmark/benchncnn.cpp

```diff
@@ -90,6 +90,13 @@ void benchmark(const char* comment, const std::vector<ncnn::Mat>& _in, const ncn
     DataReaderFromEmpty dr;
     net.load_model(dr);
 
+    // Check if model is int8 and skip on GPU
+    if (opt.use_vulkan_compute && is_model_int8(net, comment))
+    {
+        fprintf(stderr, "%20s  skipped (int8+GPU not supported)\n", comment);
+        return;
+    }
+
```

Comment:
Since only the name pattern is checked, move these lines to the beginning of `benchmark()` for skipping early without loading model, and inline `is_model_int8()` call which is simple enough.

In `fixed_path` mode, skip silently.